include/libserver/data/DataDefinitions.hpp

```diff
@@ -293,6 +295,7 @@ struct Horse
 
   dao::Field<uint32_t> luckState{0u};
   dao::Field<uint32_t> emblemUid{0u};
+  dao::Field<Clock::time_point> dateOfBirth{Clock::now()};;
```

Comment:
```suggestion
  dao::Field<Clock::time_point> dateOfBirth{};
```
do not initialize + redundant semicolor